src/main/java/us/spaceclouds42/zones/mixin/accessor/BlockUpdateS2CPacketAccessor.java

```diff
@@ -0,0 +1,19 @@
+package us.spaceclouds42.zones.mixin.accessor;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.network.packet.s2c.play.BlockUpdateS2CPacket;
+import net.minecraft.util.math.BlockPos;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(BlockUpdateS2CPacket.class)
+public interface BlockUpdateS2CPacketAccessor {
+    @Accessor
+    void setState(BlockState state);
+
+    @Accessor("state")
+    BlockState fabriczones$getState();
```

Comment:
I prefer not using `fabriczones$method` and instead just `method`

Reply:
That might (and in this case, will) conflict with methods that BlockUpdateS2CPacket already has. (or does mixin just ignore this?)

Reply:
I've never had any problems with using the same method name..

Reply:
Because one is private, one is public, and they are on different types (accessor vs the actual type)

Reply:
Member access isn't accounted for in the method signature, and accessor methods get added to the actual class.

Reply:
I've never had any issues with it and I don't like the use of the dollar sign, as that has a different purpose.